Documentation/deploy-master-single.md

```diff
@@ -112,7 +112,7 @@ spec:
     - --service-cluster-ip-range=${SERVICE_IP_RANGE}
     - --secure_port=443
     - --advertise-address=${ADVERTISE_IP}
-    - --admission-control=NamespaceLifecycle,NamespaceExists,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
+    - --admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
```

Comment:
So either don't put the kube-apiserver in the kube-system namespace -- recommended so that the service for kube-apiserver exists in the default namespace. kube-system is mainly used for add-ons, which I don't think the apiserver falls in that group. So leave NamespaceExists in place and don't put the kube-apiserver in the kube-system namespace.